src/mmc_windows.c

```diff
@@ -78,63 +41,56 @@ static int mmc_query_volume_path(const WCHAR *volume_path, struct mmc_device *de
  */
 int mmc_scan_for_devices(struct mmc_device *devices, int max_devices)
 {
-    DWORD  Error                = ERROR_SUCCESS;
-    HANDLE FindHandle           = INVALID_HANDLE_VALUE;
-    size_t Index                = 0;
-    BOOL   Success              = FALSE;
-    WCHAR  VolumeName[MAX_PATH] = L"";
+    WCHAR drive_path[MAX_PATH] = L"";
+    HANDLE drive_handle;
```

Comment:
Move these variables down their first use. I know that this isn't the typical Windows style for C code, but at least for the rest of fwup, I take advantage of C99's support for this. I prefer the style that reduces variable scope and the distance between declaration and use as much as is reasonable.